### ISSUE TITLE
HTML Files Not Created

### ISSUE THREAD

**Issue by hugoanderson5** [bedrock-viz/bedrock-viz#2]
Hello, I am trying to generate an HTML version of my world on macOS. bedrock-viz runs without error, but no HTML file is created. 

Output from command:
```
LAZERBOOK:bin troy$ ./bedrock-viz --db /Users/troy/Downloads/EileanMor/ --out /Users/troy/Downloads/EileanMorOutput/ --html-all
Reading XML from /usr/local/Cellar/bedrock-viz/v0.1.1/share/bedrock-viz/data/mcpe_viz.xml
parseLevelFile: name=/Users/troy/Downloads/EileanMor//level.dat version=8 len=2067
  Found World Spawn: x=524 y=32767 z=4
  Level name is [Eilean Mor - Copy]
DB Open: dir=/Users/troy/Downloads/EileanMor/
DB Open Status: OK (block_size=4096 bloom_filter_bits=10)
Scan keys to get world boundaries
  Bounds (chunk): DimId=0 X=(-75 267) Z=(-384 86)
  Bounds (pixel): DimId=0 X=(-1200 4272) Z=(-6144 1376) Image=(5488 7536)
  Bounds (chunk): DimId=1 X=(-35 39) Z=(-56 32)
  Bounds (pixel): DimId=1 X=(-560 624) Z=(-896 512) Image=(1200 1424)
  Bounds (chunk): DimId=2 X=(0 0) Z=(0 0)
  Bounds (pixel): DimId=2 X=(0 0) Z=(0 0) Image=(16 16)
  215899 records
Active 'hide-top', 'force-top', and 'geojson-block':
None
Parse all leveldb records
  Processing records: 10000 / 215899 (4.6%)
  Processing records: 20000 / 215899 (9.3%)
  Processing records: 30000 / 215899 (13.9%)
  Processing records: 40000 / 215899 (18.5%)
  Processing records: 50000 / 215899 (23.2%)
  Processing records: 60000 / 215899 (27.8%)
  Processing records: 70000 / 215899 (32.4%)
  Processing records: 80000 / 215899 (37.1%)
  Processing records: 90000 / 215899 (41.7%)
  Processing records: 100000 / 215899 (46.3%)
  Processing records: 110000 / 215899 (50.9%)
INFO: Unmapped remote player: 1156a42d-ca4a-35e8-9ec3-1a11bff8c441
INFO: Unmapped remote player: 5c1e4b67-8812-3270-b36c-6b8dc169284a
INFO: Unmapped remote player: 77054079-bd0b-3a74-9ced-0656aa4b975a
INFO: Unmapped remote player: 873af4fc-b420-3d62-b610-f73320a6d8f2
INFO: Unmapped remote player: 93438a90-e6f1-36a2-812e-57bdadd8eb71
INFO: Unmapped remote player: ec30c37f-177f-32b5-9e91-6c6e3f7a602f
INFO: Unmapped remote player: server_c331610d-c3b9-44c4-90ee-635c4c01b873
INFO: Unmapped remote player: server_c80a8e68-476a-40ba-8826-561c1df7d457
INFO: Unmapped remote player: server_d47a2b45-ada5-4197-b647-225919de5f65
Player Position: Dimension=0 Pos=(473.5, 64.62, -385.5 @ image 1673, 5758) Rotation=(0.000000, 0.000000)
  Processing records: 120000 / 215899 (55.6%)
  Processing records: 130000 / 215899 (60.2%)
  Processing records: 140000 / 215899 (64.8%)
  Processing records: 150000 / 215899 (69.5%)
  Processing records: 160000 / 215899 (74.1%)
  Processing records: 170000 / 215899 (78.7%)
  Processing records: 180000 / 215899 (83.4%)
  Processing records: 190000 / 215899 (88.0%)
  Processing records: 200000 / 215899 (92.6%)
  Processing records: 210000 / 215899 (97.3%)
Read 215899 records
Status: OK
Check Spawnable: Dimension 'overworld' (0)
Check Spawnable: Dimension 'nether' (1)
Check Spawnable: Dimension 'the-end' (2)
Do Output: overworld
  Generate Image
    Info: World Spawn (524, 4) is at image (1724, 6148)
  Generate Biome Image
  Generate Grass Image
  Generate Height Column Image
  Generate Height Column (grayscale) Image
  Generate Height Column (alpha) Image
  Generate Block Light Image
  Generate Sky Light Image
  Generate Slime Chunks Image
  Generate Shaded Relief Image
  Generate full-size slices
    Writing all images in one pass
    Row 0 of 7536
    Row 320 of 7536
    Row 640 of 7536
    Row 960 of 7536
    Row 1280 of 7536
    Row 1600 of 7536
    Row 1920 of 7536
    Row 2240 of 7536
    Row 2560 of 7536
    Row 2880 of 7536
    Row 3200 of 7536
    Row 3520 of 7536
    Row 3840 of 7536
    Row 4160 of 7536
    Row 4480 of 7536
    Row 4800 of 7536
    Row 5120 of 7536
    Row 5440 of 7536
    Row 5760 of 7536
    Row 6080 of 7536
    Row 6400 of 7536
    Row 6720 of 7536
    Row 7040 of 7536
    Row 7360 of 7536
Do Output: nether
  Generate Image
  Generate Biome Image
  Generate Grass Image
  Generate Height Column Image
  Generate Height Column (grayscale) Image
  Generate Height Column (alpha) Image
  Generate Block Light Image
  Generate Sky Light Image
  Generate Slime Chunks Image
  Generate Shaded Relief Image
  Generate full-size slices
    Writing all images in one pass
    Row 0 of 1424
    Row 320 of 1424
    Row 640 of 1424
    Row 960 of 1424
    Row 1280 of 1424
Do Output: the-end
  Generate Image
  Generate Biome Image
  Generate Grass Image
  Generate Height Column Image
  Generate Height Column (grayscale) Image
  Generate Height Column (alpha) Image
  Generate Block Light Image
  Generate Sky Light Image
  Generate Slime Chunks Image
  Generate Shaded Relief Image
  Generate full-size slices
    Writing all images in one pass
    Row 0 of 16
Do Output: html viewer
Copying icon images
Done.
LAZERBOOK:bin troy$ 
```

The CSS and JS files are created, but no HTML or geo coordinates file.

```
LAZERBOOK:Downloads troy$ cd EileanMorOutput
LAZERBOOK:EileanMorOutput troy$ ls -l
total 240
drwxr-xr-x  1267 troy  staff   40544 Dec 22 09:50 images
drwxr-xr-x@    9 troy  staff     288 Dec 22 09:50 js
-rw-r--r--@    1 troy  staff    3404 Dec 22 09:50 mcpe_viz.css
-rw-r--r--@    1 troy  staff  115780 Dec 22 09:50 mcpe_viz.js
```

**Comment by jasper-wan** [comment]
This is a known bug, the output filename must be specified:
```
--db /Users/troy/Downloads/EileanMor/index
```

not

```
--db /Users/troy/Downloads/EileanMor/ # error: no filename
```

The new version (v0.2) fixes this problem but it is not finished yet

**Comment by jasper-wan** [comment]
and the `ulimit` problem will also be fixed in version 0.2

**Comment by hugoanderson5** [comment]
That fixed it, thank you!
